nlp/README.md

```diff
@@ -0,0 +1,22 @@
+## DACCS - NLP docker image
+
+This image is able tu run all the python files required for the nlp notbooks.
+It has an installation of tree-tagger and java librarie needed for tree-tagger by download and extraction.
```

Comment:
this is somehow specific, and I guess this `README.md` is supposed to be a high level presentation
I would remove this line, as the user could easily read the `Dockerfile`